_includes/tutorials/produce-consume-lang/scala/markup/dev/6_0-consume-narrative.adoc

```diff
@@ -0,0 +1,56 @@
+The following Scala class will be the entry point of the consumer application.
+At `src/main/scala/io/confluent/developer/Consumer.scala` add:
+
++++++
+<pre class="snippet"><code class="java">{%
+    include_raw tutorials/produce-consume-lang/scala/code/src/main/scala/io/confluent/developer/Consumer.scala
+%}</code></pre>
++++++
+
+Lets’s describe the keys pieces of this program.
+
+NOTE: We use an externalised config and this app loads its config from the _consumer_ bloc of the `application.conf`
+
+A `Consumer#conusme` function covers most of the record consumption.
+
+[source,scala]
+----
+def consume(consumer: KafkaConsumer[Bytes, Book]): Vector[Book] = { //<1>
+
+  val books: ConsumerRecords[Bytes, Book] = consumer.poll((1 second) toJava) //<2>
+
+  books.asScala.toVector.map(_.value()) //<3>
+}
+----
+
+<1> `Consumer#conusme` takes a
+`https://kafka.apache.org/25/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html[KafkaConsumer]`
+and gives back a collection of `Book`.
+
+<2> `Consumer#consume` call the `KafkaConsumer#poll` and gets a `ConsumerRecords` filed with as much books as the
+`fetch.max.bytes` Consumer Config allows.
```

Comment:
IMO remove the dialog about fetch.max.bytes; slightly advanced and also doesn't tell the full story about how many records will be in the return set.

```suggestion
<2> We call the `KafkaConsumer#poll` function on the `KafkaConsumer` which returns a `ConsumerRecords` containing Book instances.
```